### ISSUE TITLE
SplitAndJoint Plugin need to check for null!

### ISSUE THREAD

**Issue by rsoika** [imixs/imixs-workflow#235]
```
public int run(ItemCollection adocumentContext, ItemCollection adocumentActivity) throws PluginException {

		ItemCollection evalItemCollection = ResultPlugin.evaluateWorkflowResult(adocumentActivity, adocumentContext);

```

must be checked to null!

**Comment by rsoika** [comment]
already fixed for 4.0.1

**Comment by rsoika** [comment]
also fixed for 3.9.1 

no patch needed!

**Comment by rsoika** [comment]
already fixed
